src/persist/src/storage.rs

```diff
@@ -72,40 +71,70 @@ pub trait Blob {
     fn set(&mut self, key: &str, value: Vec<u8>, allow_overwrite: bool) -> Result<(), Error>;
 }
 
-/// An implementation of [Persister] in terms of a [Buffer] and [Blob].
-pub struct StoragePersister<U: Buffer, L: Blob> {
+/// An implementation of a set of persisted streams in terms of a [Buffer] and [Blob].
```

Comment:
I'd mostly copy the old Persister docs here, too. This doesn't make as much sense to me now that it's the only impl.

Something like "/// A persister of multiplexed streams of (Key, Value, Time, Diff) updates." ?